### PR TITLE
Bionic menu clean up

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
```SUMMARY: Interface "Display name of perpetual fuel"```

#### Purpose of change
Display name of perpetual fuel
Don't display `Fuel Available:` if no fuel is available or if the current mode is `reassigning`
Replace `ostringstream` with `string`

#### Testing
With Perpetual fuel:
image

Without any Fuel:
image

### CODE REVIEW

File: src/bionics_ui.cpp [hunk 1]
```diff
@@ -59,18 +59,24 @@ static void draw_bionics_titlebar( const catacurses::window &window, player *p,
...
         for( const itype_id fuel : p->get_fuel_available( bio.id ) ) {
+            found_fuel = true;
             const item temp_fuel( fuel ) ;
             if( temp_fuel.has_flag( "PERPETUAL" ) ) {
+                fuel_string += "<color_green>" + temp_fuel.tname() + "</color>";
                 continue;
             }
-            fuel_stream << temp_fuel.tname() << ": " << "<color_green>" << p->get_value(
-                            fuel ) << "</color>" << "/" << p->get_total_fuel_capacity( fuel ) << " ";
+            fuel_string += temp_fuel.tname() + ": <color_green>" + p->get_value(
```
Comment (ZhilkinSerg) [inline src/bionics_ui.cpp]:
```suggestion
            fuel_string += temp_fuel.tname() + ": " + colorize( p->get_value(
```

File: src/bionics_ui.cpp [hunk 2]
```diff
@@ -59,18 +59,24 @@ static void draw_bionics_titlebar( const catacurses::window &window, player *p,
...
+                               fuel ) + "</color>/" + std::to_string( p->get_total_fuel_capacity( fuel ) ) + " ";
```
Comment (ZhilkinSerg) [inline src/bionics_ui.cpp]:
```suggestion
                               fuel ), c_green ) + "</color>/" + std::to_string( p->get_total_fuel_capacity( fuel ) ) + " ";
```

File: src/bionics_ui.cpp (hunk 1 above)
Comment (ZhilkinSerg) [reply src/bionics_ui.cpp]: I am not sure about syntax and style, but you should get an idea.

File: src/bionics_ui.cpp [hunk 3]
```diff
@@ -59,18 +59,24 @@ static void draw_bionics_titlebar( const catacurses::window &window, player *p,
...
-    fuel_stream << _( "Available Fuel: " );
+    std::string fuel_string;
+    bool found_fuel = false;
+    fuel_string = _( "Available Fuel: " );
     for( const bionic &bio : *p->my_bionics ) {
         for( const itype_id fuel : p->get_fuel_available( bio.id ) ) {
+            found_fuel = true;
             const item temp_fuel( fuel ) ;
             if( temp_fuel.has_flag( "PERPETUAL" ) ) {
+                fuel_string += "<color_green>" + temp_fuel.tname() + "</color>";
```
Comment (ZhilkinSerg) [inline src/bionics_ui.cpp]:
```suggestion
                fuel_string += colorize( temp_fuel.tname(), c_green );
```

**Comment by tinukedaya** [comment]
From the pictures it looks like it's hiding the bionics if you don't have the power to activate them. Is that so? 
If yes, I'd suggest instead of hiding, grey them out and put them on the bottom of the list.
If not, just ignore me...

**Comment by Fris0uman** [comment]
It's two screen shot of two different characters, the second one has no bionics.

**Pull request merged**